config/event_handling.tmpl.yml

```diff
@@ -619,13 +613,17 @@ event_groups:
     - VmConfigMissingEvent
     - NODE_NODEREADY
     - NODE_NODENOTREADY
-    - NODE_FAILEDMOUNT
-    - NODE_INVALIDDISKCAPACITY
     - POD_FAILEDVALIDATION
     - POD_DEADLINEEXCEEDED
     - POD_FAILEDSYNC
     - POD_INSUFFICIENTFREECPU
     - CONTAINER_UNHEALTHY
+    - POD_NODESELECTORMISMATCHING
+    - POD_FAILEDSCHEDULING
+    - REPLICATOR_FAILEDCREATE
```

Comment:
I think ```REPLICATOR_FAILEDCREATE``` should be under ```addition```, like ```USER_FAILED_ADD_HOST``` and ```USER_FAILED_ADD_VM_TEMPLATE```.
Same goes for ```POD_FAILEDSCHEDULING```, if we decide to put ```POD_SCHEDULED``` under ```addition```